lib/core/network/repository/profile_repository.dart

```diff
@@ -22,12 +22,13 @@ class ProfileService {
         final map = Map<String, dynamic>.from(response.data);
         return Result.value(ProfileData.fromJson(map, user.network, []));
       } else {
-        LogHelper.e('get profile status error', stacktrace: StackTrace.current);
+        LogHelper.i('get profile status error');
```

Comment:
lets pass the `server error ${response.statusMessage}` to this log

Reply:
done